ceci/main.py

```diff
@@ -123,39 +122,54 @@ def run(pipeline_config_filename, extra_config=None, dry_run=False):
     else:
         raise ValueError("Unknown pipeline launcher {launcher_name}")
 
-    # Run the pre-script.  Since it's an error for this to fail (because
```

Comment:
Almost all of this is just an indentation change, to put the main block inside the `with` statement.